ecc/bls12-377/fr/iop/utils.go

```diff
@@ -0,0 +1,378 @@
+// Copyright 2020 ConsenSys Software Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by consensys/gnark-crypto DO NOT EDIT
+
+package iop
+
+import (
+	"fmt"
+
+	"github.com/consensys/gnark-crypto/ecc/bls12-377/fr"
+	"github.com/consensys/gnark-crypto/ecc/bls12-377/fr/fft"
+)
+
+func printVector(v []fr.Element) {
+	fmt.Printf("[")
+	for i := 0; i < len(v); i++ {
+		fmt.Printf("Fr(%s), ", v[i].String())
+	}
+	fmt.Printf("]\n")
+}
+
+func printPolynomials(p []*Polynomial) {
+	fmt.Printf("[\n")
+	for i := 0; i < len(p); i++ {
+		printVector(p[i].Coefficients)
+		fmt.Printf(",\n")
+	}
+	fmt.Printf("]\n")
+}
+
+func printLayout(f Form) {
```

Comment:
add a `String()` method to the enums and a `String()` method to the `Form` type

Reply:
Yeah those were for debugging that I forgot to delete